README.md

```diff
@@ -45,43 +50,75 @@ See the [changelog](CHANGELOG.md) for the latest release. You might also find us
       **Example Path**: `/usr/lib/jvm/java-1.8.0` if `bin` exists at `/usr/lib/jvm/java-1.8.0/bin`.
     * **Note**: If the path is not set, the extension will attempt to find the path to the JDK or JRE.
 
+See [here](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Preferences.md#java-home) for more information.
```

Comment:
This information duplicates the information directly above and looks a bit strange because of the indent.
I suggest to either limit the information here more or possibly just list the requirement and reword this sentence to `See [how to set java home](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Preferences.md#java-home) for more information how this extension searches for Java.`

Reply:
Thanks for your suggestion. Fixed.